packages/core/admin/admin/src/pages/SettingsPage/pages/AuditLogs/ListView/index.js

```diff
@@ -22,6 +71,12 @@ const ListView = () => {
           defaultMessage: 'Logs of all the activities that happened on your environment',
         })}
       />
+      <ContentLayout>
+        <DynamicTable contentType="Audit logs" headers={headers} rows={data} withBulkActions>
```

Comment:
Should contentType be translated?

Reply:
no, its expected as string as its going to be a part of filter message